source/linear-algebra/source/05-GT/samples/05.ptx

```diff
@@ -0,0 +1,49 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<example xml:id="sample-GT5"><title>GT5</title>
+    <introduction>
+        <p>
+            Let <m>\mathcal{B}=\setList{\begin{bmatrix}-2\\-2\\1\end{bmatrix},\begin{bmatrix}-1\\-2\\-1\end{bmatrix},\begin{bmatrix}1\\3\\2\end{bmatrix}}</m>, and <m>\vec{v}=\begin{bmatrix}1\\2\\3\end{bmatrix}</m>.
+        </p>
+    </introduction>
+        <task>
+            <statement>
+                <p>
+                    Explain and demonstrate how to verify that <m>\mathcal{B}</m> is a basis of <m>\IR^3</m> and how to calculate <m>M_\mathcal{B}</m>, the change-of-basis matrix from the standard basis of <m>\IR^3</m> to <m>\mathcal{B}</m>.
+                </p>
+            </statement>
+            <solution>
+            <p>
+                We can accomplish both tasks by calculating the RREF of the following matrix:
+                <me>\RREF\left[\begin{array}{ccc|ccc}-2&amp;-1&amp;1&amp;1&amp;0&amp;0\\
+                        -2&amp;-2&amp;3&amp;0&amp;1&amp;0\\1&amp;-1&amp;2&amp;0&amp;0&amp;1\end{array}\right]
+                    =
+                \left[\begin{array}{ccc|ccc}1&amp;0&amp;0&amp;1&amp;-1&amp;1\\
+                    0&amp;1&amp;0&amp;-7&amp;5&amp;-4\\
+                    0&amp;0&amp;1&amp;-4&amp; 3&amp;-2\end{array}\right].</me>
```

Comment:
This is a fancy way of saying "find the inverse of the appropriate matrix". Haven't read the activities yet but would be interesting maybe to make that connection?